src/main/java/Rabbit/RabbitException/DeleteException.java

```diff
@@ -0,0 +1,24 @@
+package Rabbit.RabbitException;
+
+public class DeleteException extends RabbitException {
+    public enum Type {
+        FORMAT, INDEX;
+    }
+    private Type type;
+
+    public DeleteException(Type type) {
+        this.type = type;
+    }
+
+    @Override
+    public String toString() {
+        switch(this.type) {
+            case FORMAT:
+                return "Type 'delete index' to delete a task if that's what you want.";
+            case INDEX:
```

Comment:
Your switch statements for AddToListException were in the right format. Perhaps you missed out this part while editing?